components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/constant/OrganizationManagementConstants.java

```diff
@@ -80,7 +80,8 @@ public class OrganizationManagementConstants {
     public static final List<String> ALL_ORGANIZATION_PERMISSIONS = Collections.unmodifiableList(Arrays
             .asList(CREATE_ORGANIZATION_PERMISSION, VIEW_ORGANIZATION_PERMISSION, UPDATE_ORGANIZATION_PERMISSION,
                     DELETE_ORGANIZATION_PERMISSION));
-    public static final String ROOT_TENANT_DOMAIN = "RootTenantDomain";
```

Comment:
Why we deleted this constant ?

Reply:
This was removed by mistake. Reverted by PR:https://github.com/wso2/identity-organization-management-core/pull/155